multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -512,6 +526,43 @@ def transform_image(
 
         return image_features, image_types
 
+    def transform_document(
+        self,
+        df: pd.DataFrame,
+    ) -> Tuple[Dict[str, List[List[str]]], Dict[str, str]]:
+        """
+        Preprocess document data by collecting their paths together. If one sample has multiple documents
```

Comment:
What if they are not separated by `;` or has other corner cases of input, do we have any error handling logic?

Reply:
The current version actually does not support multiple documents, it's just a copy docstring. But I will investigate how we should handle this case.